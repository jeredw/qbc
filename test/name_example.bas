    INPUT "Old Name: "; OldFN$
    INPUT "New Name: "; NewFN$
    NAME OldFN$ AS NewFN$
