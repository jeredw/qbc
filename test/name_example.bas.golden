(program 
  (statement 
    (input_statement INPUT "Old Name: " ; 
      (variable_or_function_call OldFN$))) \n 
  (statement 
    (input_statement INPUT "New Name: " ; 
      (variable_or_function_call NewFN$))) \n 
  (statement 
    (name_statement NAME 
      (expr 
        (variable_or_function_call OldFN$)) AS 
      (expr 
        (variable_or_function_call NewFN$)))) \n <EOF>)
